modules/system/classes/UpdateManager.php

```diff
@@ -533,11 +533,14 @@ public function updatePlugin($name)
     }
 
     /**
-     * Removes an existing plugin
+     * Rollback an existing plugin
+     *
      * @param string $name Plugin name.
+     * @param null $stopOnVersion If this parameter is specified, the process stops after
+     * the specified version is rolled back.
      * @return self
      */
-    public function rollbackPlugin($name)
+    public function rollbackPlugin($name, $stopOnVersion = null)
```

Comment:
We're going to start implementing type hints where possible. Could you make both arguments be type hinted to `string` and the return type defined as `self`.
```suggestion
    public function rollbackPlugin(string $name, string $stopOnVersion = null): self
```